docs/source/test_network.md

```diff
@@ -261,6 +264,9 @@ the chaincode on the channel specified using the channel flag (or `mychannel`
 if no channel is specified).  If you are deploying a chaincode for the first
 time, the script will install the chaincode dependencies. You can use the
 language flag, `-l`, to install the Go, typescript or javascript versions of the chaincode.
+
+Before installing the Go, typescript or javascript version of the chaincode make sure that
+the required language compiler is installed on you local enviroment.
```

Comment:
Ok, thanks for moving the guidance up higher in the file. This section can be removed now.